src/tasks/processBulkUploadTask.ts

```diff
@@ -243,6 +245,60 @@ const loadTaskRunnerAuthContext = async (): Promise<AuthContext> => ({
 	},
 });
 
+/**
+ * A value for initializing and incrementing the recursion guard
+ * @see getDetailsFromError
+ */
+const ONE = 1;
+/**
+ * A limit for the recursion guard below
+ * @see getDetailsFromError
+ */
+const ONE_THOUSAND = 1000;
```

Comment:
I believe the intention of no magic numbers is for the named variable to reference the point of the value, so this should be `MAX_RECURSION = 1000`